File: auto-docs/test_executables/matlab/basic-box-plot.m
signin('theengineear', 'o9zlr0hy6z')

y0 = randn(50,1);
y1 = randn(50,1)+1;
trace1 = struct(...
  'y', y0, ...
  'type', 'box');
trace2 = struct(...
  'y', y1, ...
  'type', 'box');
data = {trace1, trace2};

response = plotly(data, struct('filename', 'basic-box-plot', 'fileopt', 'overwrite', 'auto_open', 'false'));
plot_url = response.url

File: auto-docs/test_executables/python/basic-box-plot.py
import plotly.plotly as py
from plotly.graph_objs import *

py.sign_in('theengineear', 'o9zlr0hy6z')

import numpy as np
y0 = np.random.randn(50)
y1 = np.random.randn(50)+1
trace1 = Box(
    y=y0
)
trace2 = Box(
    y=y1
)
data = Data([trace1, trace2])

if 'fig' not in locals():
    if 'data' not in locals():
        raise Exception('no data OR figure!!')
    fig = dict(data=data)  # assumes fig or data
if 'layout' not in fig:
    fig['layout'] = dict()
if 'margin' not in fig['layout']:
    fig['layout']['margin'] = dict(t=50, b=50, r=50, l=50)
fig['layout'].update(autosize=False, width=500, height=500)


plot_url = py.plot(data, filename='basic-box-plot', auto_open=False)